activities/value-and-impact/all-press.md

```diff
@@ -0,0 +1,31 @@
+---
+type: resource
+---
+
+
+# All press and media mentions of the Foundation for Public Code
+
+Here's the complete list of all mentions of the Foundation for Public Code on 3rd party platforms.
```

Comment:
"third" would be easier to read. Is there a reason for using numbers here?

Reply:
Nope! Happy to change.